sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```diff
@@ -166,13 +161,24 @@ export class MsalOpenBrowser extends MsalNode {
         }
       }
 
-      const abortSignal = options?.abortSignal;
-      if (abortSignal) {
-        abortSignal.addEventListener("abort", () => {
+      app.on("connection", (socket) => socketToDestroy.push(socket));
+
+      app.on("listening", () => {
+        const openPromise = this.openAuthCodeUrl(scopes);
+
+        const abortSignal = options?.abortSignal;
+        if (abortSignal) {
+          abortSignal.addEventListener("abort", () => {
+            cleanup();
+            reject(new Error("Aborted"));
+          });
+        }
+
+        openPromise.then().catch((e) => {
           cleanup();
           reject(new Error("Aborted"));
         });
```

Comment:
Aborted or throw the error that is caught?

Reply:
I’ll just throw the error I receive!